tools/apiview/emitters/typespec-apiview/src/apiview.ts

```diff
@@ -396,6 +401,18 @@ export class ApiView {
         this.tokenize(obj.elementType);
         this.punctuation("[]");
         break;
+      case SyntaxKind.ArrayLiteral:
```

Comment:
do you have support for `const` too or those wouldn't show up/cause an issue

Reply:
Can you give me an example? I'd be happy to add it.

Reply:
```
const a = 123;
const b = #{name: "abc"};
const c = a;

@decorator(a, b)
model foo {}
```

Reply:
Hmm... it looks like constants are not part of the TypeSpec `Namespace` model, so I'm not sure I should display them? I can access them through the `NamespaceNode.statements` property though.

Reply:
yeah its the same as `alias` they are not something an standard emitter should worry about. So if you don't need to show them its fine just wnated to make sure it won't crash

Reply:
I'll opt to release this now since it's blocking people. Constants just "won't show up" right now but that's probably not right either. But I'd prefer to add that as a separate PR.

Reply:
For tracking: https://github.com/Azure/azure-sdk-tools/issues/8586

Reply:
> yeah its the same as `alias` they are not something an standard emitter should worry about. So if you don't need to show them its fine just wnated to make sure it won't crash

But we _do_ show aliases in APIView, so we should show these too.